pkg/routeagent_driver/constants/constants.go

```diff
@@ -27,8 +27,8 @@ const (
 	InputChain         = "INPUT"
 	ForwardChain       = "FORWARD"
 	MangleTable        = "mangle"
-	RemoteCIDRIPSet    = "SUBMARINER-REMOTECIDRS"
```

Comment:
@tpantelis , Do you think we should flush/remove also sets with previous names in uninstall ?